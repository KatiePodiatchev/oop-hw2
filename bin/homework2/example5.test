# Should add edges to the graph

# Create a graph
CreateGraph graph1

# Create some nodes and add them to the graph
CreateNode n1 5
CreateNode n2 5
CreateNode n3 10
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add some edges
AddEdge graph1 n1 n2
AddEdge graph1 n2 n3
AddEdge graph1 n3 n1

# Add Edge from node to itself
AddEdge graph1 n1 n1

# Add Edge twice
AddEdge graph1 n2 n3

# Add Edge to the other direction
AddEdge graph1 n3 n2

# Find a path
FindPath graph1 n1 -> n1
FindPath graph1 n1 -> n3
FindPath graph1 n3 -> n1
FindPath graph1 n3 n1 -> n1 n3
