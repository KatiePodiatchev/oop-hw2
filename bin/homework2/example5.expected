# Should add edges to the graph

# Create a graph
created graph graph1

# Create some nodes and add them to the graph
created node n1 with cost 5
created node n2 with cost 5
created node n3 with cost 10
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add some edges
added edge from n1 to n2 in graph1
added edge from n2 to n3 in graph1
added edge from n3 to n1 in graph1

# Add Edge from node to itself
added edge from n1 to n1 in graph1

# Add Edge twice
added edge from n2 to n3 in graph1

# Add Edge to the other direction
added edge from n3 to n2 in graph1

# Find a path
shortest path in graph1: n1
shortest path in graph1: n1 n2 n3
shortest path in graph1: n3 n1
shortest path in graph1: n1
